sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2667 @@
+!> @file
+!! @brief Process vertical profile climatologic data for WAM.
+!!
+!! This file contains all data need to form exosphere and used
+!! whole atmsopheric modeling (wam). It has two modules and some
+!! routines to compute the temperature and compositions of neutral
+!! density in specific values.
+!! The original package contains fortran 77 blockdata and common statement
+!! they are all recoded to use modules and use-statements.
+!!
+!! @author Hann-Ming Henry Juang  NCEP/EMC
+!-----------------------------------------------------------------------
+   MODULE WAM_GTD7BK_MOD
+!          MSISE-00 01-FEB-02                                           
+!x----COMMON/PARM7/
+      REAL ::      PT1(50),PT2(50),PT3(50),PA1(50),PA2(50),PA3(50),     &
```

Comment:
We need each of these to be declared on its own line, and each to have a doxygen comment. Like this:

```
REAL :: PT1(50)  !< Some description of PT1.
REAL :: PT2(50)  !< Some description of PT2.
etc.

```

All module variables need a doxygen comment.

Also, where did "50" come from? Can you use a named constant instead?